src/app/components/modals/modal-delete/modal-delete.component.scss

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Digital Excellence Copyright (C) 2020 Brend Smits
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Lesser General Public License as published
+ *   by the Free Software Foundation version 3 of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty
+ *   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *   See the GNU Lesser General Public License for more details.
+ *
+ *   You can find a copy of the GNU Lesser General Public License
+ *   along with this program, in the LICENSE.md file in the root project directory.
+ *   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+ */
+
+@import "assets/styles/variables";
+
+.modal-body{
+  position: unset;
+}
+
+.modal-delete-container {
+  background: #f8f8f8;
+  width: 600px;
+  border-radius: 6px;
+  border: none;
+  outline: none;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16);
+  padding: 15px;
+
+  button.close{
+    border: none;
+    background: none;
+    display: flex;
+    float: right;
+    &:focus, &:active{
+      border:none;
+      outline:none;
+    }
+  }
+}
+
+.delete-form {
+  margin-top: 20px;
+  input[type="date"] {
+    border: none;
+    border-radius: 6px;
+    box-shadow: 0 3px 6px #0000001a;
+    height: 45px;
+    width: 100%;
+    margin-bottom: 25px;
+    padding: 0 15px;
+    &:focus {
+      outline: none;
+      border: 1px solid #707070;
+    }
+  }
+
+  p.date-error-message {
+    color: $accent-color-primary;
+  }
+
+  &__checkbox {
+    display: flex;
+    align-items: center;
+    margin-top: 10px;
+    input[type="checkbox"] {
+      border: solid 1px #707070;
+      margin-right: 15px;
+    }
+
+    h4 {
+      margin: 0;
+    }
+  }
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: flex-end;
+
+  button:nth-child(1) {
+    margin-right: 15px;
+  }
+}
+
+.highlight-wrapper {
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+
+  #neverEndingCheckbox, #checkbox {
+    width: 16px;
+    height: 16px;
+    margin-right: 10px;
+  }
+
+  label {
+    margin-bottom: 0px;
+  }
+
+  .date-spacing {
+    margin-left: 15px;

```

Comment:
This should be changed, it will mess up formatting like: https://gyazo.com/802f6686a1d1a28bbbc5c52f269899d1